crates/dbsp/src/circuit/tokio.rs

```diff
@@ -0,0 +1,7 @@
+use once_cell::sync::Lazy;
+use tokio::runtime::Runtime;
+
+/// The process running dbsp can share a single Tokio runtime.
+///
+/// This is `pub` so it can be used in adapters too for IO.
+pub static TOKIO: Lazy<Runtime> = Lazy::new(|| Runtime::new().unwrap());
```

Comment:
I couldn't find in the docs how many threads this will create. As many as CPU cores?

Reply:
Yup: https://docs.rs/tokio/latest/src/tokio/runtime/builder.rs.html#357